consensus/ibft/ibft.go

```diff
@@ -679,12 +682,12 @@ func (i *Ibft) writeTransactions(gasLimit uint64, transition transitionInterface
 		// no errors, pop the tx from the pool
 		i.txpool.Pop(tx)
 
-		successful = append(successful, tx)
+		transactions = append(transactions, tx)
 	}
 
-	i.logger.Info("picked out txns from pool", "num", len(successful), "remaining", i.txpool.Length())
+	i.logger.Info("picked out txns from pool", "num", len(transactions), "remaining", i.txpool.Length())
```

Comment:
Please refactor this log to be clear on how many transactions failed, and how many succeeded

Reply:
Great catch! Fixed 79a49accce1b9344e439ec8fe560eccfae0392c1